internal/providers/terraform/azure/testdata/application_gateway_test/application_gateway_test.golden

```diff
@@ -0,0 +1,24 @@
+
+ Name                                         Monthly Qty  Unit              Monthly Cost 
+                                                                                          
+ azurerm_application_gateway.standard                                                     
+ ├─ Gateway usage (basic,small)                     1,460  hours                   $36.50 
+ ├─ Data processing (0-10TB)                       10,240  GB                      $81.92 
+ ├─ Data processing (10-40TB)                      30,720  GB                     $245.76 
+ └─ Data processing (over 40TB)                    59,040  GB                     $472.32 
+                                                                                          
+ azurerm_application_gateway.wafV2                                                        
+ ├─ Gateway usage (WAF v2)                          1,460  hours                  $525.60 
+ └─ V2 capacity units (WAF)                        10,000  CU                     $144.00 
+                                                                                          
+ azurerm_application_gateway.withoutUsage                                                 
+ ├─ Gateway usage (WAF v2)                          1,460  hours                  $525.60 
+ ├─ V2 capacity units (WAF)                Monthly cost depends on usage: $0.0144 per CU  
+ └─ Data processing (0-10TB)               Monthly cost depends on usage: $0 per GB       
```

Comment:
We trying to be consistent with order with cloud provider pricing page, so can u move the logic to next order:
- Gateway usage
- Data processing
- V2 capacity units